SmartDeviceLink/SDLFileManager.m

```diff
@@ -179,8 +179,13 @@ - (void)didEnterStateFetchingInitialList {
             BLOCK_RETURN;
         }
 
-        // If there was an error, we'll pass the error to the startup handler and cancel out
+        // If there was an error, we'll pass the error to the startup handler and cancel out other then in case of DISALLOWED
         if (error != nil) {
+            // In the case we are DISALLOWED we still want to transition to a ready state
+            if([error.localizedDescription isEqualToString:@"DISALLOWED"]) {
```

Comment:
Needs a log, and see below for passing along the `resultCode`.

Reply:
added